crypto/rand/rand_win.c

```diff
@@ -688,7 +688,7 @@ static void readscreen(void)
     int h;                      /* screen height */
     int y;                      /* y-coordinate of screen lines to grab */
     int n = 16;                 /* number of screen lines to grab at a time */
-    BITMAPINFOHEADER bi;        /* info about the bitmap */
+    BITMAPINFO bi;              /* info about the bitmap */
```

Comment:
Why change the type?

Reply:
see recent mail to openssl-security; it appears the type is wrong in some cases, windows adds RGB data.

Reply:
Oh, right, I recall seeing that...  but then, the CI failures indicate that we'd need to have a closer look at the structure...